### PR TITLE
Drinking singulo ignores supermatter hallucinations and pulls nearby objects

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
Drinking a singulo will now:

- Give immunity to supermatter hallucinations
- Pulls objects to you based on the total volume in your system (20u = 1x1, 45u = 2x2, 80u = 3x3)
- Makes a burp and supermatter rays/sound when objects are pulled

The new ingredient is:

- Vokda 5u 
- Wine 5u
- Liquid Dark Matter 1u (replaces Radium)

## Why It's Good For The Game
More cool effects for drinks.  Singularity is all about gravity and the drink should have a theme around that.

dreamseeker_2q21YXS698

## Changelog
:cl:
add: Drinking singulo will now ignore supermatter hallucinations and pull objects to you
balance: Change singulo drink recipe to require liquid dark matter instead of radium.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
>Negate floating in zero gravity (can move normally on floors, just like magboots)

makes no sense and is stupid

**Comment by bluetshirtguy** [comment]
Can't we just have a cool animated drink with no strings attached?

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1874,12 +1900,6 @@
 	taste_description = "a wall of bricks"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/glass_style/drinking_glass/bacchus_blessing
-	required_drink_type = /datum/reagent/consumable/ethanol/bacchus_blessing
-	name = "Bacchus' Blessing"
-	desc = "You didn't think it was possible for a liquid to be so utterly revolting. Are you sure about this...?"
-	icon_state = "glass_brown2"
-
```
Comment (Time-Green) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:1882]: Why is this being renoved?

**Comment by moocowswag** [comment]
How often does this pull people too you, (I very often use dark matter for my bombs and stuff and most of the time its pulling people to me and this might be pretty big for my playstyle)

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (timothymtorres) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Fixed with 51b688baf4ab006ba56aa3c25d003ffcad88542a thanks for pointing it out!

**Comment by timothymtorres** [comment]
> Does this only pull one time or multiple types as you ingest it?

There is a small chance that occurs about once a minute.

> And if its over time How often does this pull things too you, (I very often use dark matter for my bombs and stuff and most of the time its for the purpose of pulling people to me and this might be pretty big for my playstyle)

The pull strength is based on how much of the drink is in your system.  Normal dark matter reactions can reach up to 6 tiles radius (full screen) but I felt that was too powerful for the drink effect so I lowered it to a max radius of 3 tiles.

**Pull request merged**